graphql_compiler/tests/test_schema_fingerprint.py

```diff
@@ -175,6 +175,23 @@ def test_directive_location_order(self):
         """
         _compare_schema_fingerprints(self, schema_text1, schema_text2)
 
+    def test_argument_order_of_field_definition_directive(self):
+        schema_text1 = """
+            directive @custom_directive(b: String, a: Int) on FIELD_DEFINITION
+
+            type Foo {
+                my_field: Int @custom_directive(b: "123", a: 1)
+            }
+        """
+        schema_text2 = """
+            directive @custom_directive(b: String, a: Int) on FIELD_DEFINITION
```

Comment:
Should this be `b, a` order or `a, b` order? Might be worth digging into the GraphQL spec to see if there's a reason not to reorder here.

Reply:
I intentionally set this as `b, a` order. I already have a test called `test_directive_definition_argument_order` to test the order of arguments in a directive definition.

Reply:
Did that answer your question? I am a bit confused of what you are trying to get at.